app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -1,52 +1,30 @@
 <template>
   <div class="pure-g">
     <div class="pure-u-1-2 gobierto-data-summary-header">
-      <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-clock"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelUpdated }}
-        </span>
-        <span
-          v-if="dateUpdated"
-          class="gobierto-data-summary-header-container-text"
-        >
-          {{ dateUpdated | convertDate }}
-        </span>
-      </div>
-      <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-calendar"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelFrequency }}
-        </span>
-        <span
-          v-if="frequencyDataset"
-          class="gobierto-data-summary-header-container-text"
-        >
-          {{ frequencyDataset }}
-        </span>
-      </div>
-      <div class="gobierto-data-summary-header-container">
-        <i
-          class="fas fa-tag"
-          style="color: var(--color-base); opacity: .25"
-        />
-        <span class="gobierto-data-summary-header-container-label">
-          {{ labelSubject }}
-        </span>
-        <a
-          v-if="categoryDataset"
-          href=""
-          class="gobierto-data-summary-header-container-text-link"
-        >
-          {{ categoryDataset }}
-        </a>
-      </div>
+      <InfoBlockText
+        v-if="dateUpdated"
+        icon="clock"
+        icon-color="var(--color-base)"
+        opacity=".25"
+        :label="labelUpdated"
+        :text="dateUpdated | convertDate"
+      />
+      <InfoBlockText
+        v-if="frequencyDataset"
+        icon="calendar"
+        icon-color="var(--color-base)"
+        opacity=".25"
+        :label="labelFrequency"
+        :text="frequencyDataset"
+      />
+      <InfoBlockText
+        v-if="categoryDataset"
+        icon="tag"
+        icon-color="var(--color-base)"
```

Comment:
Look at the previous comment about the fallback color, I think these are a good examples